CampfireSDK/src/main/java/com/sayzen/campfiresdk/compose/publication/post/pages/TablePage.kt

```diff
@@ -9,15 +10,27 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
+import com.dzen.campfire.api.models.images.ImageRef
 import com.dzen.campfire.api.models.publications.post.PageTable
 import com.sayzen.campfiresdk.compose.util.TableWithBorders
+import com.sayzen.campfiresdk.support.load
+import com.sup.dev.android.libs.image_loader.ImageLoader
+import com.sup.dev.android.libs.screens.navigator.Navigator
+import com.sup.dev.android.views.screens.SImageView
 import sh.sit.bonfire.formatting.compose.buildInlineAnnotatedString
 import sh.sit.bonfire.formatting.core.BonfireFormatter
 import sh.sit.bonfire.images.RemoteImage
 
 @Composable
 internal fun PageTableRenderer(page: PageTable) {
     val colors = MaterialTheme.colorScheme
+    val images = mutableListOf<ImageRef>()
+    var imageIndex = 0
+
+    for (cell in page.cells) {
+        if (cell.image.isEmpty()) continue
+        images.add(cell.image)
+    }
```

Comment:
```suggestion
    val images = remember(page) {
        val ret = mutableListOf<ImageRef>()
        for (cell in page.cells) {
            if (cell.image.isEmpty()) {
                continue
            }
            images.add(cell.image)
        }
        ret.toList()
    }

     var imageIndex = 0
```
^ Не проверял. Короче надо через `remember` сделать.